RELEASING.md

```diff
@@ -10,64 +9,54 @@ and v2.0). There only exists a single patch version (or pre-release
 alphas, betas and release candidates) for each major/minor version. When
 we do a new patch release, the updates are made in-place.
 
-
 ## Preparing for a release
-
 1. Ensure master docs are passing semaphore tests.
-
 2. Check out latest master locally.
 
 ## Creating the release
-
 ### When doing a major/minor release ONLY
-
-These steps are for creating a new major/minor release, with the expectation that a release candidate will be created, published and tested before the final release is announced.
+These steps are for creating a new major/minor release, with the expectation 
+that a release candidate will be created, published and tested before the final 
+release is announced.
 
 1. Run release-scripts/do_release.py to copy master directory to the new version directory
 2. Add the new version to `_data/versions.yaml` (add the date of the release in the description)
    - Add new versions that are a release candidate to the end of the file.
      If a release is added to the top of the file it will become the 'Latest Release'.
 3. Add a section in `_config.yaml` so that `page.version` will be set correctly in the new subdirectory:
-
-   ```
+````
    -
      scope:
        path: vX.Y
      values:
        version: vX.Y
-   ```
-
+````
 4. Update the `RELEASE_STREAM` variable at the top of calico_node/Makefile to the new major/minor version.
-
-4. Test the changes locally then open a pull request, make sure it passes CI, then merge.
-
-5. run `make -C calico_node release` to create container images for calico/node. Push them by following the instructions that the make command prints.
-
+2. Test the changes locally then open a pull request, make sure it passes CI and get it reviewed.
```

Comment:
Yeah, because why not :smile: The numbers don't mean anything, just that they should be rendered as a numbered list.